.github/workflows/ci-experimental.yml

```
@@ -61,27 +61,19 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-        with:
-          submodules: recursive
 
-      - name: Get submodule versions
+      - name: Get Submodules Hash
```

Pure nitpick from my side, but can we get these names to follow the same capitalisation (i.e. only first letter capitalised)?

```suggestion
      - name: Get submodules hash
```

Here and elsewhere.

---

Pardon my morning ignorance. Fixed